core/matrix/dense.cpp

```diff
@@ -425,10 +431,9 @@ void Dense<ValueType>::move_to(Dense<next_precision<ValueType>> *result)
 template <typename ValueType>
 void Dense<ValueType>::convert_to(Coo<ValueType, int32> *result) const
 {
-    conversion_helper(
-        result, this,
-        dense::template make_convert_to_coo<const Dense<ValueType> *&,
-                                            decltype(result)>);
+    conversion_helper(result, this, [](auto &in, auto &out) {
```

Comment:
This change was necessary since the compiler was for some reason not able to deduce which overload of make_convert_to_coo to use, even though we specified all template parameters.

Reply:
Could you also provide the error message?

Reply:
```
I'm guessing this has to do that the return type of make_* is `auto`, which in turn is based on the return type of the templated function make_register_operation.
```